packages/govuk-frontend/src/govuk/components/notification-banner/notification-banner.test.js

```diff
@@ -1,165 +1,219 @@
 const { renderAndInitialise, goToComponent } = require('govuk-frontend-helpers/puppeteer')
 const { getExamples } = require('govuk-frontend-lib/files')
 
-describe('Notification banner, when type is set to "success"', () => {
+describe('Notification banner', () => {
   let examples
 
   beforeAll(async () => {
     examples = await getExamples('notification-banner')
   })
 
-  it('has the correct tabindex attribute to be focused with JavaScript', async () => {
-    await goToComponent(page, 'notification-banner', {
-      exampleName: 'with-type-as-success'
+  describe('when type is set to "success"', () => {
```

Comment:
Nice little test refactor with the `describe()` nesting, mind if we split it into another commit?

Reply:
No probs, I'll split the extra `describe`s in their own commit to separate them from the implementation of the error throwing.